index.html

```diff
@@ -40,21 +41,186 @@ <h3>I'm a Software Developer.</h3>
+                    <li><a href="#"><img src="./images/icon4.png" alt="AngelList-icon"></a></li>
+                    <li><a href="#"><img src="./images/icon3.png" alt="Twitter-icon"></a></li>
+                    <li><a href="#"><img src="./images/icon5.png" alt="Medium-icon"></a></li>
                 </ul>
             </div>
 
 
 
         </section>
+    
+        
+    
+        
+        <section  class= "work-container">
+            <h2 class="title"> My Recent Works</h2>
+            <div class="feature-project">
+                
+                <div class="phimg">
+                <img src="./images/placeholder.png" alt="placeholder-img">
+                </div>
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                     industry's standard dummy text ever since the 1500s, when an unknown printer took a 
+                     standard dummy text.
+                </p>
+
+                <div class="languages">
+                    <a href="#" class="language">css</a>
+                    <a href="#" class="language"> html</a>
+                    <a href="#" class="language">bootstrap</a>
+                    <a href="#" class="language"> ruby</a>
+                </div>
+
+                <a href="#">See Project</a>
+            </div>
+
+                <div class="projects">
+                    <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
+                        
+                            <a href="#" class="project-btn">See project</a>
+                        
+                        
+                     </div>
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <a href="#" class="project-language"> html</a>
+                            <a href="#" class="project-language">bootstrap</a>
+                            <a href="#" class="project-language"> ruby</a>
+                        </div>
```

Comment:
- [ ] Let us wrap our technologies inside the `ul li tag` as well as increase the size of it to look the same we have on the figma.
The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li